mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -24,26 +24,31 @@ def __init__(
 
     def get_signed_conformity_scores(
         self,
+        X: ArrayLike,
         y: ArrayLike,
         y_pred: ArrayLike,
     ) -> NDArray:
         """
-        Compute the signed conformity scores from the predicted values
-        and the observed ones, from the following formula:
+        Compute the signed conformity scores from the observed values
+        and the predicted ones, from the following formula:
```

Comment:
QUESTION: Is there a added value in this change? If not, I suggest to not make change.